pkg/contextstore/store_test.go

```diff
@@ -0,0 +1,119 @@
+package contextstore
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+func TestStoreInitScratch(t *testing.T) {
+	dirname, err := ioutil.TempDir("", "test-store-init-scratch")
+	assert.NilError(t, err)
+	defer os.RemoveAll(dirname)
+	testDir := filepath.Join(dirname, "test")
+	_, err = NewStore(testDir)
+	assert.NilError(t, err)
+	metaFS, err := os.Stat(filepath.Join(testDir, metadataDir))
+	assert.NilError(t, err)
+	assert.Equal(t, metaFS.IsDir(), true)
+	tlsFS, err := os.Stat(filepath.Join(testDir, tlsDir))
+	assert.NilError(t, err)
+	assert.Equal(t, tlsFS.IsDir(), true)
```

Comment:
This is `assert.Assert` (also it is the other way round from your previous use of this pattern, I guess one of them is backwards).

I won't comment on all the potential places for `assert.Assert` to be used around here.